satellite/console/service.go

```diff
@@ -280,15 +339,15 @@ func (s *Service) SetDefaultPaymentMethod(ctx context.Context, projectPaymentID
 }
 
 // DeleteProjectPaymentMethod deletes selected payment method
-func (s *Service) DeleteProjectPaymentMethod(ctx context.Context, projectPayment uuid.UUID) (err error) {
+func (s *Service) DeleteProjectPaymentMethod(ctx context.Context, projectPaymentID uuid.UUID) (err error) {
```

Comment:
Detach or smth like this?

Reply:
We are deleting database record from project payment methods. I think that in this case naming is correct